### DISCUSSION TITLE
Review of Mendeley

### DISCUSSION THREAD

**Discussion by AlexeyMatveev686** [ONLYOFFICE/onlyoffice.github.io#236] [Polls]
<img src="https://raw.githubusercontent.com/ONLYOFFICE/onlyoffice.github.io/master/discussions-icons/mendeley.svg">

**Comment by jbsbbruno** [comment]
**The plugin doesn't work!** An error occurs when logging into the Mendeley website. After configuring the API, it is **not possible to log in to the account** to synchronize References. 

Fedora 39 user linux kernel 6.5.12-300.fc39.x86_64. OnlyOffice version 7.5.1.23 (flatpak)

**Reply by AlexeyMatveev686** [reply]
Hello, if you are using the desktop version, this plugin does not work. We are aware of this, it is written in the [Readme ](https://github.com/ONLYOFFICE/onlyoffice.github.io/tree/master/sdkjs-plugins/content/mendeley#known-issues)of the plugin. If you have another problem, please give more information.

**Comment by Kidplayer666** [comment]
A shame for it not to work on the desktop version

**Reply by Misokados** [reply]
Hello,
we aware of this, and we're working on it.

**Comment by wol-existss** [comment]
It is very hard to recommend this, as its functionality is limited on desktop versions.
